modules/alloy/src/main/ts/ephox/alloy/behaviour/disabling/DisableState.ts

```diff
@@ -4,12 +4,12 @@ import { nuState } from '../common/BehaviourState';
 import { DisableState } from './DisableTypes';
 
 const init = (): DisableState => {
-  const lastDisabledState = Cell(false);
+  const lastDisabledState = Cell<boolean | undefined>(undefined);
```

Comment:
This could be a singleton
```suggestion
  const lastDisabledState = Singleton.value<boolean>();
```
Singetons don't need to be initialized. Their `get` method returns an optional though, so you could use `get().getOr(false)`. You can also see if it contains a value with `isSet`

Reply:
Thanks, made changes to the `DisableApi` to return `Optional<boolean>` too.